turtlesim/include/turtlesim/turtle.h

```diff
@@ -56,60 +58,58 @@ namespace turtlesim
 class Turtle
 {
 public:
-  Turtle(const ros::NodeHandle& nh, const QImage& turtle_image, const QPointF& pos, float orient);
+  Turtle(rclcpp::Node::SharedPtr& node_handle, std::string& real_name, const QImage& turtle_image, const QPointF& pos, float orient);
 
   bool update(double dt, QPainter& path_painter, const QImage& path_image, qreal canvas_width, qreal canvas_height);
   void paint(QPainter &painter);
-private:
-  void velocityCallback(const geometry_msgs::Twist::ConstPtr& vel);
-  bool setPenCallback(turtlesim::SetPen::Request&, turtlesim::SetPen::Response&);
-  bool teleportRelativeCallback(turtlesim::TeleportRelative::Request&, turtlesim::TeleportRelative::Response&);
-  bool teleportAbsoluteCallback(turtlesim::TeleportAbsolute::Request&, turtlesim::TeleportAbsolute::Response&);
 
+private:
   void rotateImage();
 
-  ros::NodeHandle nh_;
+  rclcpp::Node::SharedPtr nh_;
 
   QImage turtle_image_;
   QImage turtle_rotated_image_;
 
   QPointF pos_;
-  qreal orient_;
+  float orient_;
```

Comment:
These types don't have to change either. Same as https://github.com/ros/ros_tutorials/pull/53#discussion_r292097587

Reply:
I've replace float to qreal in turtle.h and turtle.cpp